src/server/dcp_server/models.py

```diff
@@ -286,7 +304,28 @@ def eval(self, img):
         
         return final_mask
 
+class CellClassifierShallowModel:
+
+    def __init__(self, model_config, train_config, eval_config):
 
+        self.model_config = model_config
```

Comment:
you are not using self.model_config for anything? How about RandomForestClassifier parameters?

Reply:
For now, I use it for everything, so I just added "model_class" in classifier model init and use default RandomForest values.